carma-messenger-core/cpp_message/src/cpp_message.cpp

```diff
@@ -40,6 +41,8 @@ namespace cpp_message
         mobility_operation_message_sub_=nh_->subscribe<>("outgoing_plain_mobility_operation_message",5, &Message::outbound_mobility_operation_message_callback,this);
         mobility_response_message_pub_=nh_->advertise<cav_msgs::ByteArray>("incoming_mobility_response_message_decoded",5);
         mobility_response_message_sub_=nh_->subscribe<>("outgoing_plain_mobility_operation_message",5, &Message::outbound_mobility_response_message_callback,this);
+        mobility_path_message_pub_=nh_->advertise<cav_msgs::ByteArray>("incoming_mobility_path_message_decoded",5);
```

Comment:
I think the message type on this is wrong. Shouldn't it be cav_msgs/MobilityPath ?

Reply:
This node should only publish raw binary to the outbound_binary_msg topic. All the other publications should be cav_msgs or j2735_msgs types.

Reply:
Yes, you're right, it should be cav_msgs/MobilityPath